src/plugins/kibana_react/public/url_template_editor/url_template_editor.tsx

```diff
@@ -125,8 +125,19 @@ export const UrlTemplateEditor: React.FC<UrlTemplateEditorProps> = ({
     };
   }, [variables]);
 
+  const { euiTheme } = useEuiTheme();
+  const editorStyle = css({
+    '.monaco-editor .lines-content.monaco-editor-background': {
+      margin: `0 ${euiTheme.size.s}`,
+    },
+  });
+
   return (
-    <div className={'urlTemplateEditor__container'} onKeyDown={handleKeyDown}>
+    <div
+      data-test-subj="url-template-editor-container"
+      className={editorStyle}
```

Comment:
I'm not entirely sure we're not able to leverage `@emotion/react` and set the `css` property on the `div`.

Reply:
Should you be using `import { css } from '@emotion/react'` instead of `import { css } from '@emotion/css'`? When I tried that, it works 🤔